jest.config.js

```diff
@@ -1,44 +1,44 @@
 module.exports = {
-  'collectCoverage': true,
-  'collectCoverageFrom': [
+  collectCoverage: true,
+  collectCoverageFrom: [
     '**/*.js',
     '!**/*.test.js'
   ],
-  'coverageDirectory': 'test-output',
-  'coverageReporters': [
+  coverageDirectory: 'test-output',
+  coverageReporters: [
     'text-summary',
     'cobertura',
     'lcov'
   ],
-  'coveragePathIgnorePatterns': [
+  coveragePathIgnorePatterns: [
     'jest.config.js',
     'index.js',
     '<rootDir>/node_modules/',
     '<rootDir>/test-output/',
     '<rootDir>/test/',
     '.*/__mocks__/.*'
   ],
-  'coverageThreshold': {
-    'global': {
-      'statements': 89
+  coverageThreshold: {
+    global: {
+      statements: 89

```

Comment:
I'm getting a test coverage of 85%, so this will not build as it doesn't pass the 89% test threshold.

Reply:
That's odd. I'm seeing 84% for functions and 64% for branches but 89% for statements, which the threshold applies to. The pipeline shows the same results and passed the build.